Employees/Services/IEmployee.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Employees.Entites;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Employees.Services
```

Comment:
A couple of points on naming here - `IEmployee` is not actually the employee, but a way to store several employees somewhere so a better name might be `IEmployeeRepository` or `IEmployeeStore`. If you had an interface called `IEmployee` I'd expect to see classes like `PermanentEmployee` implementing it as they're a type of employee.

Secondly, interfaces should try not to name methods after the first implementation. You have `WriteToFile` here, but the next implementation might write to a database or a cache. Just `Write` or `WriteEmployees` would be a better choice.